### DISCUSSION TITLE
Browser Prefixing

### DISCUSSION THREAD

**Discussion by AdamChlan** [gregsullivan/_tw#72] [General]
Hi @gregsullivan thank you for the great theme. I'm curious to understand what level of browser prefixing / support comes out of the box. The TW docs indicate that if using the TW CLI, browser prefixing is a part of the build. I am seeing a bit of this in my built css file. But if we needed to expand the support for other browsers, would we want to look at autoprefixer for this?

**Comment by gregsullivan** [comment]
Hi Adam—Thanks for posting this!

I've generally used Tailwind's defaults for browser prefixing, as its support matrix aligns with what I promise to my clients. That said, my understanding is that Tailwind supports modifying the supported browsers via the `browserslist` key in `package.json`. You can see that key being taken into account in the code powering Tailwind's CLI:

https://github.com/tailwindlabs/tailwindcss/blob/e4b32a7a32e5844649fff4132ddb9a1675558629/src/cli/build/plugin.js#L47

You'll find documentation on how to format the `browserslist` key here:

https://github.com/browserslist/browserslist

Hope that helps! If you find you're not getting the prefixes you expect, you shouldn't encounter any issues adding `autoprefixer` to the `postcss.config.js` file. (It used to be included with _tw by default, before that functionality migrated into the CLI itself.)

I'm going to convert this to a discussion, but please let me know if you have any other questions!

**Reply by AdamChlan** [reply]
This is very helpful, thank you! I will plan on looking into the browserlist config.

**Reply by gregsullivan** [reply]
No problem at all! If you're able to check back in and let me know how it all turned out, I'd love to hear about it.
